packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -137,6 +137,7 @@ export function CrudMoreActionsMenu({ slotProps, overallActions, selectiveAction
                 {!!overallActions?.length && (
                     <CrudMoreActionsGroup
                         groupTitle={<FormattedMessage id="comet.crudMoreActions.overallActions" defaultMessage="Overall actions" />}
+                        typographyProps={selectiveActions?.length ? undefined : { sx: { display: "none" } }}
```

Comment:
Please change this to not render groupTitle at all instead of hiding it with CSS.